integration/pom.xml

```diff
@@ -202,6 +202,16 @@
             <artifactId>spring-boot-test</artifactId>
             <version>${spring.boot.version}</version>
             <scope>test</scope>
+            <exclusions>
```

Comment:
looks like this changes are from 6.3 not 6.2 ?

Reply:
@KatrukOV Is this relevant? It seems ok in my smoke test and doesn't seem to cause issues. Have you tested these changes with Refsite? I am ok with merging this if alex is ok with this.

Reply:
Yes, this is relevant, we checked it.

Reply:
I'm ok with this change